fastlane/testflight.md

```diff
@@ -72,15 +76,13 @@ Log into your GitHub account to create a personal access token; this is one of t
 
 This is the second one of two GitHub secrets needed for your build.
 
-The first time you build with the GitHub Browser Build method for any DIY app, you will make up a password and record it as `MATCH_PASSWORD`. Note, if you later lose `MATCH_PASSWORD`, you will need to delete and make a new Match-Secrets repository (next step).
+The first time you build with the GitHub Browser Build method for any DIY app, you will make up a password and record it as `MATCH_PASSWORD`. Note, if you later lose `MATCH_PASSWORD`, you will need to delete and make a new Match-Secrets repository (next step). You use the same password for all DIY apps.
 
 ## Setup GitHub Match-Secrets Repository
 
-The creation of the Match-Secrets repository is a common step for all GitHub Browser Builds; do this step only once. You must be logged into your GitHub account.
-
-1. Create a [new empty repository](https://github.com/new) titled `Match-Secrets`. It should be private.
+> This step is no longer required, your Match-Secrets repository is automatically created the first time you run a GitHub Action. You can skip ahead to [Setup Github Trio repository](#setup-github-trio-repository).
```

Comment:
Perhaps just remove this `## Setup GitHub Match-Secrets Repository` section then? Or write that it is created automatically.

Reply:
I modified testflight.md and fixed this, added a few links and fixed a few typos.